scio-core/src/main/scala/com/spotify/scio/util/ScioUtil.scala

```diff
@@ -106,15 +107,23 @@ private[scio] object ScioUtil {
 
   def defaultFilenamePolicy(
     path: String,
```

Comment:
IMHO `path` must be complete and either point to a file or a directory. It should not contain a `prefix` part